discord/ext/pages/pagination.py

```diff
@@ -1090,8 +1090,10 @@ async def edit(
 
         if page_content.custom_view:
             self.update_custom_view(page_content.custom_view)
-
-        self.user = user or self.user
+        if user or self.user:
+            self.user = user or self.user
+        else:
+            self.usercheck = False
```

Comment:
Need to test if this still works in the same logic, but it's cleaner:
```suggestion
        if user:
            self.user = user
        elif not self.user:
            self.usercheck = False
```

Reply:
I'll test it soon